lib/chitter.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'pg'
+require './spec/database_helper.rb'
+require 'date'
+class Chitter
+  attr_accessor :content, :time
+  def initialize(content, time)
+    @content = content
+    @time = time.chomp(':00')
+  end
+
+  def self.all
+    connection = if ENV['RACK_ENV'] == 'test'
```

Comment:
Good set up of db connection, try putting this in a separate file and having it run before each test, saves a lot of time having to repeat yourself